spec/unit/client/decision/apply_to_spec.rb

```diff
@@ -174,6 +210,40 @@ class Decision
           end
         end
 
+        describe "private#path" do
```

Comment:
Is there a reason why we're testing a private method instead of testing the expected behavior like on L:39?

(I now realize I introduce a bad pattern by using `send` in this test instead of writing the expected url)